easybuild/easyconfigs/a/Alpha/Alpha-20190607-foss-2019b-Python-2.7.16.eb

```diff
@@ -0,0 +1,82 @@
+easyblock = 'Bundle'
+
+name = 'Alpha'
+local_commit = 'e027eb0fea1e'
+version = '20190607'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.lirmm.fr/~swenson/alpha/alpha.htm'
+description = "Alpha is a tool designed for detailed comparative study of bacteriophage genomes."
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '2.7.16'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Biopython', '1.75', versionsuffix),
+    ('BLAST+', '2.9.0'),
+    ('Clustal-Omega', '1.2.4'),
+    ('GenomeTools', '1.6.1'),
+    ('GTK+', '3.24.13'),
+    ('matplotlib', '2.2.4', versionsuffix),
+    ('networkx', '2.2', versionsuffix),
+    ('pydot', '1.4.1'),
+    ('PyGObject', '3.34.0', versionsuffix),
+    ('pygraphviz', '1.5'),
+    ('python-igraph', '0.8.0'),
+]
+
+components = [
+    ('pyinclude', '0f8a00b8c10c', {
+        'easyblock': 'Tarball',
+        'source_urls': ['https://bitbucket.org/thekswenson/%(name)s/get/'],
+        'sources': [{
+            'download_filename': '%(version)s.zip',
+            'filename': '%(name)s-%(version)s.zip',
+            'extract_cmd': "unzip %s && mv thekswenson-%(name)s-%(version)s %(name)s"
+        }],
+        'checksums': ['9dc9b2bafbfe7a694ff4025164dc6e356989ee89b914691457fb0a474fe66c9e'],
+    }),
+    (name, local_commit, {
+        'easyblock': 'Tarball',
+        'source_urls': ['https://bitbucket.org/thekswenson/%(namelower)s/get/'],
+        'sources': [{
+            'download_filename': '%(version)s.zip',
+            'filename': '%(namelower)s-%(version)s.zip',
+            'extract_cmd': "unzip %s && mv thekswenson-%(namelower)s-%(version)s %(namelower)s"
+        }],
+        'patches': ['%s-%s_fix-email.patch' % (name, version)],
+        'checksums': [
+            'a9a0857ee080864c0d187f1f537aebdb3a61e58dd555b97dffbf2c2d73465f2a',  # alpha-e027eb0fea1e.zip
+            '342d5d37bcfc19a2487c3b08860adc4246736919948b1968f899b0c595d7125d',  # Alpha-20190607_fix-email.patch
+        ],
+    }),
+    # PythonPackage is last to not have its source copied by Tarball blocks
```

Comment:
Can you clarify the need to have this listed last? No objections, just wondering what's going on...

Reply:
The problem was that the `tarball` easyblock does not like to be used as component of a bundle. On each iteration it deletes and re-creates `installdir`, destroying any previous installation. So I resorted to put the PythonPackage after the Tarballs.

However, this was kind of hacky, so I made some improvements to the `tarball` easyblock that allow using it as a component more easily. See https://github.com/easybuilders/easybuild-easyblocks/pull/1989